README.md

```diff
@@ -152,6 +160,23 @@ If you only wish to change the text without changing attributes you can also pas
 </div>
 ```
 
+### @includeCached
+
+Just like [`@include`](https://laravel.com/docs/11.x/blade#including-subviews) but cached. Everything returned will be cached with [`Cache::flexible()`](https://laravel.com/docs/11.x/cache#swr) for 5 minutes; and refreshed in the background until it expires after 24 hours. After that it will be refreshed as it's normally working. The cache key is a combination of the view name and the current slugified url. That way this can be used with multisite setups:
```

Comment:
```suggestion
Just like [`@include`](https://laravel.com/docs/11.x/blade#including-subviews) but cached. Everything returned will be cached with [`Cache::flexible()`](https://laravel.com/docs/11.x/cache#swr) for 5 minutes; and refreshed in the background until it expires after 24 hours. After that it will be refreshed as usual. The cache key is a combination of the view name and the current slugified url. That way this can be used with multisite setups:
```